src/api/index.js

```diff
@@ -4,7 +4,17 @@ axios.defaults.baseURL = 'https://api-dev.tipcat.net';
 axios.defaults.headers.common['Authorization'] = `Bearer ${localStorage.getItem('token')}`;
 
 export const fetchers = {
-  //members
+  // SignUp
+  
+
+  // Members
   getMember: () => axios.get('/api/members/current'),
-  getMamberUsingPOST: () => axios.post('/api/members/current'),
+  createMember: () => axios.post('/api/members/current'),
+  getMemberById: ({accountId, memberId}) => axios.post(`/api/accounts/${accountId}/members/${memberId}`),
+  updateMember: ({accountId, id, ...data}) => axios.put(`/api/accounts/${accountId}/members/${id}`, data),
```

Comment:
There is `memberId` in one method and `id` in another. Thy could be more consistent.